### PR TITLE
ARSN-238: ignore phd keys with no versions

### PR DESCRIPTION
**Pull request by ghost**
Please see [this comment](https://scality.atlassian.net/browse/ARTESCA-5165?focusedCommentId=318554) for details

### CODE REVIEW

**Comment by ghost** [comment]
ping

**Review by KillianG — APPROVED** [review]

File: lib/storage/metadata/mongoclient/MongoClientInterface.js [hunk 1]
```diff
@@ -1379,8 +1379,15 @@ class MongoClientInterface {
                             const key = obj.key.slice(DB_PREFIXES.Master.length);
                             getLatestVersion(c, key, BUCKET_VERSIONS.v1, log, (err, version) => {
                                 if (err) {
+                                    // ignoring phd keys with no versions as the all versions
```
Comment (francoisferrand) [inline lib/storage/metadata/mongoclient/MongoClientInterface.js]: typo in the comment:
```
// ignoring phd keys with no versions as the all versions
                                         ^^^^^^^
```

**Review by welansari — APPROVED** [review]

**Review by ghost — APPROVED** [review]
LGTM

**Review by anurag4DSB — APPROVED** [review]
might be worth documenting, this behavior.

**Review by anurag4DSB — CHANGES_REQUESTED** [review]
Putting a blocker here, code looks good
But as discussed the code should be in 8.1 as the bug is for ARTESCA.

**Pull request closed without merging**